webdriver/tests/bidi/script/call_function/strict_mode.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from webdriver.bidi.modules.script import ContextTarget, ScriptEvaluateResultException
+from ... import any_int, any_string, recursive_compare
+from .. import any_stack_trace, specific_error_response
+
+
+@pytest.mark.asyncio
+async def test_strict_mode(bidi_session, top_context):
+    with pytest.raises(ScriptEvaluateResultException) as exception:
+        await bidi_session.script.call_function(
+            function_declaration="()=>{'use strict';return x=1}",
+            await_promise=False,
+            target=ContextTarget(top_context["context"]),
+        )
+
+    recursive_compare(specific_error_response({"type": "error"}), exception.value.result)
+
+    result = await bidi_session.script.call_function(
+        function_declaration="()=>{return y=1}",
+        await_promise=False,
+        target=ContextTarget(top_context["context"]),
+    )
+
+    assert result == {
+        "type": "number",
+        "value": 1}
+
+    with pytest.raises(ScriptEvaluateResultException) as exception:
+        await bidi_session.script.call_function(
+            function_declaration="()=>{'use strict';return z=1}",
```

Comment:
What's the difference to the first test with `x=1`? I assume this is just duplicated?

Reply:
With the changes it looks fine now. Before it was really just duplicated code given that the script tried to set `z` instead of `x` in strict mode as the last step. Under such a condition it was failing again, and now it will not. Thanks for getting this updated.